File: cpp/gabaritos/gabarito5.cpp
#include <iostream>

int main() {
    int nums[] = {3, 1, 4, 1, 5, 9, 2};
    int tamanho = sizeof(nums) / sizeof(nums[0]);
    int maximo = nums[0];
    for (int i = 1; i < tamanho; i++) {
        if (nums[i] > maximo) {
            maximo = nums[i];
        }
    }
    std::cout << "O maior número é: " << maximo << std::endl;
    return 0;
}

File: csharp/gabaritos/gabarito5.cs
using System;
using System.Collections.Generic;

class Program
{
    static void Main()
    {
        List<int> numeros = new List<int> { 3, 1, 4, 1, 5, 9, 2 };
        int maximo = numeros[0];
        foreach (int num in numeros)
        {
            if (num > maximo)
            {
                maximo = num;
            }
        }
        Console.WriteLine($"O maior número é: {maximo}");
    }
}